products/memcached.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Memcached
+category: db
+iconSlug: memcached
+permalink: /memcached
+versionCommand: memcached -h
+activeSupportColumn: true
```

Comment:
Does memcached follow a two phase support ?

1. first active support, with bug and security fixes,
2. then security support, with critical security fixes only.

Is this policy documented somewhere ?

Reply:
Ok, would prefer if this column was removed then. We can always add it back later if we have more information.